install-parser.sh

```diff
@@ -1,5 +1,5 @@
 set -eu
-PARSER="stanford-corenlp-full-2015-12-09"
+PARSER="stanford-corenlp-full-2016-10-31"
```

Comment:
There is a known bug (see https://github.com/stanfordnlp/CoreNLP/issues/241) with version 3.6.0 and unquoted null characters in the JSON response. The easiest fix was to upgrade to 3.7.0. Otherwise we should probably implement a post-parsing sanity check to scrub invalid characters, since null bytes cause postgresql errors.